source/includes/_endpoint-variable.md

```diff
@@ -217,7 +224,7 @@ omitted, one will be generated from the required "name".
 You may also include "values", which will create the column of data
 corresponding to this variable definition. See [Importing Data: Column-by-column](#column-by-column) for details and examples.
 
-You may instead also include an "expr" to derive a variable as a function of
+You may instead also include an "derivation" to derive a variable as a function of
```

Comment:
Did the API change? "expr" has been what the body attribute is called on POST /variables/ to create a derived variable.